Cloudinary/Operation/02_CloudinaryMapping.md

```diff
@@ -0,0 +1,135 @@
+The mapping procedure of a Cloudinary attribute is identical to the mapping of any other attribute which is also described in the *DataHub* documentation, see [Manage the ETL mappings](../../DataHub/Operation/01_ManageETLMappings.md).
+Nevertheless, the possible mappings of a Cloudinary attribute, both, from an external source or to an external source is described in this chapter below.   
+
+
+## Map a cloudinary source attribute
+
+Map a source attribute set with a Cloudinary attribute to a certain destination attribute set, for instance to transfer the Cloudinary files from the PIM products to an external web shop or marketplace.
+
+#### Prerequisites
+
+- At least one ETL attribute set mapping has been created, see [Create an ETL mapping](../../DataHub/Operation/01_ManageETLMappings.md#create-an-ETL-mapping).
+- The Cloudinary attribute has been assigned to the source attribute set of the mapping.
+
+#### Procedure
+
+*DataHub > Settings > Tab ETL*
+
+![Attribute set mappings](../../Assets/Screenshots/DataHub/Settings/ETL/AttributeSetMappings.png "[Attribute set mappings]")
+
+> [Info] The following procedure describes exemplary how to map a Cloudinary image attribute. The procedure is largely identical to map a Cloudinary video attribute. The differences are explicitly specified at the appropriate place.
+
+1. Click the attribute set mapping you want to edit in the list of attribute set mappings.   
+  The *Mapping from source set to destination set* view is displayed.
+
+  ![Mapping](../../Assets/Screenshots/DataHub/Settings/ETL/Mapping.png "[Mapping]")
+
+2. Click the row with the destination attribute to which you want to map the Cloudinary image attribute. The destination attribute must have either the *String* data type or the *Cloudinary image* data type.  
+  The *Settings* section is displayed on the right side.
+
+  ![Cloudinary source attribute](../../Assets/Screenshots/DataHub/Settings/ETL/CloudinarySourceAttribute.png "[Cloudinary source attribute]")
+
+  > [Info] Depending on the selected row, the fields displayed in the *Settings* section differ:
+  - If the selected row contains only a destination attribute, the destination attribute as well as the *Extension* and *Override change tracking mode* drop-down lists are displayed.
+  - If the selected row contains an attribute mapping with source, destination attribute and the extension, the destination attribute as well as the *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are displayed. The *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are preset. If the selected extension requires further configuration settings, the *Configuration* section is displayed below the *Settings* section.  
+
```

Comment:
Die Info Zeile ist als Info (graufarbe) angezeigt, aber die Aufzählungszeilen nicht. Ist es nur ein Problem der Darstellung in GitHub? S. https://github.com/Actindo-AG/Documentation/blob/ACD-581/Cloudinary/Operation/02_CloudinaryMapping.md